app/javascript/dashboard/i18n/locale/en/contact.json

```
@@ -385,5 +385,53 @@
     "DROPDOWN_ITEM": {
       "ID": "(ID: {identifier})"
     }
+  },
+
+  "COMPOSE_NEW_CONVERSATION": {
+    "CONTACT_SEARCH": {
+      "ERROR_MESSAGE": "Could not search for contacts. Please try again later."
+    },
+    "FORM": {
+      "GO_TO_CONVERSATION": "View",
+      "SUCCESS_MESSAGE": "Conversation created successfully",
+      "ERROR_MESSAGE": "Could not create conversation. Please try again later.",
+      "NO_INBOX_ALERT": "Couldn't find an inbox to initiate a new conversation with this contact.",
+      "CONTACT_SELECTOR": {
+        "LABEL": "To :",
```

```suggestion
        "LABEL": "To",
```

Let us add the colons in the code.